orbit.sh

```diff
@@ -146,8 +144,15 @@ while [[ $# -gt 0 ]]; do
             ;;
         # run the formatter over the repository
         -f|--format)
+            if ! command -v pre-commit &>/dev/null; then
+                echo "[INFO] Installing pre-commit..."
+                pip install pre-commit
```

Comment:
Wondering if this should be:
```bash
python_exe=$(extract_isaacsim_python)
# install the rl-frameworks specified
${python_exe} -m pip install pre-commit
```

This way we don't install stuff to a user's local or global python site-packages?

Reply:
Hmm, I am not sure. I will leave that up to you maintainers to decide.

One question, though. Would a contributor then need to extend `PATH`/`PYTHONPATH` (or source Isaac Sim python env) in order to `pre-commit run`? If so, it might complicate things too much.